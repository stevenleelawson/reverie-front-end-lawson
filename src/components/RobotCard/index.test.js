import React from 'react';
import { shallow } from 'enzyme';
import { RobotCard } from './index';

describe('RobotCard', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<RobotCard />)
    expect(wrapper).toMatchSnapshot();
  });
});
