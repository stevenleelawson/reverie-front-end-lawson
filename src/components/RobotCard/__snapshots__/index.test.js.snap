// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RobotCard should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1 />,
        <p>
          height: 
          <span
                    className="taco"
                    contentEditable="true"
                    onBlur={[Function]}
          />
</p>,
        <p>
          weight: 
          <span
                    contentEditable="true"
          />
</p>,
        <p>
          intelligence: 
</p>,
        <p>
          date added: 
</p>,
        <p>
          first active: 
</p>,
        <button
          className="deactivate"
          onClick={[Function]}
>
          Deactivate Robot
</button>,
      ],
      "className": "card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "height: ",
            <span
              className="taco"
              contentEditable="true"
              onBlur={[Function]}
/>,
          ],
        },
        "ref": null,
        "rendered": Array [
          "height: ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": "taco",
              "contentEditable": "true",
              "onBlur": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": "span",
          },
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "weight: ",
            <span
              contentEditable="true"
/>,
          ],
        },
        "ref": null,
        "rendered": Array [
          "weight: ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "contentEditable": "true",
            },
            "ref": null,
            "rendered": null,
            "type": "span",
          },
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "intelligence: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "intelligence: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "date added: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "date added: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "first active: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "first active: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Deactivate Robot",
          "className": "deactivate",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": "Deactivate Robot",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1 />,
          <p>
            height: 
            <span
                        className="taco"
                        contentEditable="true"
                        onBlur={[Function]}
            />
</p>,
          <p>
            weight: 
            <span
                        contentEditable="true"
            />
</p>,
          <p>
            intelligence: 
</p>,
          <p>
            date added: 
</p>,
          <p>
            first active: 
</p>,
          <button
            className="deactivate"
            onClick={[Function]}
>
            Deactivate Robot
</button>,
        ],
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "height: ",
              <span
                className="taco"
                contentEditable="true"
                onBlur={[Function]}
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "height: ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
                "className": "taco",
                "contentEditable": "true",
                "onBlur": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": "span",
            },
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "weight: ",
              <span
                contentEditable="true"
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "weight: ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
                "contentEditable": "true",
              },
              "ref": null,
              "rendered": null,
              "type": "span",
            },
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "intelligence: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "intelligence: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "date added: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "date added: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "first active: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "first active: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Deactivate Robot",
            "className": "deactivate",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "Deactivate Robot",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
